README.md

```diff
@@ -211,3 +211,11 @@ Target alignment refers to the alignment of the pivot relative to the target of
 ### Root Alignment
 
 Root alignment refers to the alignment of the tooltip's root relative to the pivot. See `rootAlign` and `rootAlignOffset`.
+
+## Clamping
+
+By default the tooltip is "clamped" to its parent. Meaning even if the target leaves the viewport, the tooltip would never leave the view port. The clamping behaviour can be controlled via the `clamp` property.
```

Comment:
`viewport` is one word. Might also want to replace `viewport` with `parent`; not sure on the nomenclature we're using here.